### PR TITLE
Update kws20.py

### PR DESCRIPTION
**Pull request by batuhan-gundogdu**
The STFT-based (librosa.effects) stretching is replaced by the time-domain stretching

### CODE REVIEW

File: datasets/kws20.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
 
 import librosa
```
Comment (aniktash) [inline datasets/kws20.py:39]: you may want to remove import librosa if not needed anymore

Comment (batuhan-gundogdu) [reply datasets/kws20.py:39]: librosa is still needed for loading the audio

**Review by aniktash — COMMENTED** [review]
Changes OK. Linter complains, pls check.

**Review by rotx-maxim — CHANGES_REQUESTED** [review]
Please do not change requirements.txt in the submodule. If possible, please combine the three PRs into one.

**Comment by rotx-maxim** [comment]
Merged into #53

**Pull request closed without merging**